clients/go/internal/embedded/data/VERSION

```diff
@@ -1 +1 @@
-1.0.4-SNAPSHOT
+1.0.5
```

Comment:
❓ Is this correct?

Reply:
Nope! Does this means I also added 1.0.5 as the next development version for the Java modules as well? :see_no_evil:

Reply:
Actually not - the poms show 1.0.5-SNAPSHOT, it's strange that the Go one removed the SNAPSHOT tag...Good catch. I'll look into it :thinking: